yeonjae/baekjoon/수학/2869-달팽이는올라가고싶다.js

```diff
@@ -0,0 +1,24 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const [climb, drop, treeHeight] = input[0].split(" ").map(Number);
+  const date = Math.ceil((treeHeight - drop) / (climb - drop));
+
+  console.log(date);
```

Comment:
```suggestion
  // 풀이
  const [climb, drop, treeHeight] = input[0].split(" ").map(Number);
  const result = Math.ceil((treeHeight - drop) / (climb - drop));

  console.log(result);
```
- 풀이잘봤습니다!! 🙇‍♂️
- data라는 변수명보다는 결과를 의미하는 result도 좋을 것 같아요 :)

Reply:
넵! 코멘트감사합니당!